nextflow.config

```diff
@@ -34,6 +34,16 @@ params {
   save_reference = null // Built Indexes not saved
   sequencing_center = null // No sequencing center to be written in BAM header in MapReads process
   sentieon = null // Not using Sentieon by default
+  
+  // options: Trimming
+  trim_fastq = false
+  clip_r1 = 0
+  clip_r2 = 0
+  three_prime_clip_r1 = 0
+  three_prime_clip_r2 = 0
+  trim_nextseq = 0
+  skip_trimming = false
```

Comment:
Dont `trim_fastq` and `skip_trimming` do the same thing in opposite scenarios? Maybe just pick one?